frontend/src/components/MainHeader.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="main-header">
+    <header class="header">
+      <div class="container">
+        <a target="_blank" href="https://woowacourse.github.io/"
+          ><img class="logo" src="/assets/logo/logo_full_dark.png"
+        /></a>
+        <h3 class="title"><router-link to="/recruits">지원</router-link></h3>
```

Comment:
```suggestion
        <a target="_blank" href="https://woowacourse.github.io/">
          <img class="logo" src="/assets/logo/logo_full_dark.png" />
        </a>
        <h3 class="title">
          <router-link to="/recruits">지원</router-link>
        </h3>
```
prettier가 막 똑똑하진 않네요😁 이렇게 하면 더 잘보일려나요?